apps/src/componentLibrary/segmentedButtons/SegmentedButtons.story.tsx

```diff
@@ -0,0 +1,310 @@
+                ...values,
+                [componentArg.selectedButtonValue]: value,
+              }))
+            }
+          />
+        </div>
+      ))}
+    </>
+  );
+};
+
+export const DefaultSegmentedButtons = SingleTemplate.bind({});
+DefaultSegmentedButtons.args = {
+  buttons: [
+    {label: 'Label', value: 'label'},
+    {
+      label: 'Another label',
+      value: 'another-label',
+    },
+    {
+      label: 'Text',
+      value: 'text',
+    },
+  ],
+  size: 'm',
+  selectedButtonValue: 'label',
+  onChange: value => console.log('clicked: ', value),
+};
+
+export const DisabledSegmentedButtons = MultipleTemplate.bind({});
+DisabledSegmentedButtons.args = {
+  components: [
+    {
+      buttons: [
+        {
+          label: 'Label',
+          disabled: true,
+          value: 'label',
+        },
+        {
+          label: 'Another label',
+          value: 'another-label',
+        },
+        {
+          label: 'Text',
+          value: 'text',
+        },
+      ],
+      size: 'm',
+      selectedButtonValue: 'another-label',
+      onChange: value => console.log('clicked: ', value),
+    },
+  ],
+};
+
+export const SizesOfSegmentedButtons = MultipleTemplate.bind({});
+SizesOfSegmentedButtons.args = {
+  components: [
+    {
+      buttons: [
+        {label: 'Label', value: 'label-xs'},
+        {
+          label: 'Another label',
+          value: 'another-label-xs',
+        },
+        {
+          label: 'Text',
+          value: 'text-xs',
+        },
+      ],
+      size: 'xs',
+      selectedButtonValue: 'label-xs',
+      onChange: value => console.log('clicked: ', value),
+    },
+    {
+      buttons: [
+        {label: 'Label', value: 'label-s'},
+        {
+          label: 'Another label',
+          value: 'another-label-s',
+        },
+        {
+          label: 'Text',
+          value: 'text-s',
+        },
+      ],
+      size: 's',
+      selectedButtonValue: 'another-label-s',
+      onChange: value => console.log('clicked: ', value),
+    },
+    {
+      buttons: [
+        {label: 'Label', value: 'label-m'},
+        {
+          label: 'Another label',
+          value: 'another-label-m',
+        },
+        {
+          label: 'Text',
+          value: 'text-m',
+        },
+      ],
+      size: 'm',
+      selectedButtonValue: 'text-m',
+      onChange: value => console.log('clicked: ', value),
+    },
+    {
+      buttons: [
+        {label: 'Label', value: 'label-l'},
+        {
+          label: 'Another label',
+          value: 'another-label-l',
+        },
+        {
+          label: 'Text',
+          value: 'text-l',
+        },
+      ],
+      size: 'l',
+      selectedButtonValue: 'label-l',
+      onChange: value => console.log('clicked: ', value),
+    },
+  ],
+};
+
+export const TypesOfSegmentedButtons = MultipleTemplate.bind({});
+TypesOfSegmentedButtons.args = {
+  components: [
+    {
+      buttons: [
+        {label: 'Label', value: 'label'},
```

Comment:
Nit: 'Label one' for consistency.

Reply:
Oh, that's one is actually intended, so that one who'll be checking storybook will be able to see how one word `_SegmentedButton` will look like